cdk/lib/braze-components.ts

```diff
@@ -20,11 +21,16 @@ export class BrazeComponents extends GuStack {
     constructor(scope: App, id: string, props: BrazeComponentsStackProps) {
         super(scope, id, props);
 
-        const bucketNameFromStaticCloudformationStack = 'braze-components-storybook';
+        // Get bucket name from SSM Parameter Store
+        const bucketNameParameter = StringParameter.fromStringParameterName(
+            this,
+            'StorybookAssetsBucketParameter',
+            '/braze-components/storybook-assets-bucket',
```

Comment:
the format of bucket keys (in membership account - but targeting is similar) is like this
<img width="349" height="144" alt="image" src="https://github.com/user-attachments/assets/55826bc5-f245-4629-a5ef-bf08393d95bc" />

I'm not sure why we go for "." instead of "-" - I think that's a riffraff thing, but should we be consistent?
```suggestion
            '/braze-components/storybook-assets.bucket',
```